core/themes/seven/style.css

```diff
@@ -181,13 +181,20 @@ ul.menu li.expanded {
 /**
  * Tabs.
  */
+@media only screen and (max-width: 600px) {
+  .tabs {
+    clear: both;
+  }
+}
+
```

Comment:
Normally we attempt to do mobile-first CSS. So `clear: both` would be outside a media query and `clear: none` would be in a `min-width: 600px` media query. However, I think this is understandable in this situation. Are the other media queries in Seven using the same breakpoint? If it's cleaner one way or the other, let's go the cleanest route. If it's the same either way, let's try to use breakpoints to target higher resolution screens instead of lower resolution ones.